src/components/BackEndRolePage/BackEndBrag.js

```diff
@@ -8,9 +9,11 @@ import QuoteBubble from '../QuoteBubble';
 const QUOTE = 'Yesterday I was implementing PUT method for missing RESTful API endpoint.';
 
 function BackEndBrag() {
+  const { gender } = useContext(GenderContext);
```

Comment:
I think actually instead of passing gender to this character wrapper and extracting context in route we can handle it directly inside each character.
Therefore get this context inside Fox, Robot and Whale characters directly.